.github/workflows/ci.yml

```diff
@@ -0,0 +1,138 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    strategy:
+      # Allows for matrix sub-jobs to fail without canceling the rest
+      fail-fast: false
+
+      # MATRIX:
+      # =======
+      # Required parameters:
+      #  os                  the os to run on
+      #  python-version      the python version to use
+      #  backend             the backend to use
+      #  env                 any additional env variables. Set to '{}' for none
+      # Optional parameters:
+      #  extra_hash          extra hash str to differentiate from other caches with similar name (must always start with '-')
+      matrix:
+        # Tests [amd64]
+        #
+        # FIXME: 'cpp' tests seems to fail due to compilation errors (numpy_pythran_unit)
```

Comment:
To fix this is Travis I've temporarily pinned the Pythran version to 0.9.7: https://github.com/cython/cython/commit/8609e0fa7d361f1392823ff6e1a618720cd62df3. It's probably worth doing that here too. Obviously it'll go when the issue with Pythran is properly fixed

Reply:
I did this in my last commit earlier today (it was causing merge conflicts) :)